content/configuration/externals.md

```diff
@@ -3,15 +3,19 @@ title: Externals
 contributors:
   - sokra
   - skipjack
+  - pksjce
 ---
 
-?> Description
+`externals` configuration in webpack provides a way of not including a dependency in the bundle. Instead the created bundle relies on that dependency to be present in the consumers environment.
+This typically applies to library developers though application developers can make good use of this feature too.
 
 ## `externals`
 
 `string` `regex` `function` `array` `object`
```

Comment:
Using it with `regex` is not explained later in this document. Could you either explain it or add a TODO note?